pkg/model/gcemodel/firewall.go

```diff
@@ -124,6 +124,7 @@ func (b *FirewallModelBuilder) Build(c *fi.ModelBuilderContext) error {
 		}
 		if b.NetworkingIsCilium() {
 			t.Allowed = append(t.Allowed, fmt.Sprintf("udp:%d", wellknownports.VxlanUDP))
+			t.Allowed = append(t.Allowed, fmt.Sprintf("tcp:%d", wellknownports.EtcdCiliumClientPort))
```

Comment:
Should only be given if `model.UseCiliumEtcd(b.Cluster)` or somesuch.

Reply:
Sure, done.